src/Microsoft.WSMan.Management/WsManHelper.cs

```diff
@@ -1098,10 +1098,10 @@ internal static void LoadResourceData()
                         string Line = _sr.ReadLine();
                         if (Line.Contains("="))
                         {
-                            string[] arr = Line.Split(new char[] { '=' }, 2);
+                            string[] arr = Line.Split('=', count: 2);
                             if (!ResourceValueCache.ContainsKey(arr[0].Trim()))
                             {
-                                string value = arr[1].TrimStart(new char[] { '"' }).TrimEnd(new char[] { '"' });
+                                string value = arr[1].TrimStart('"').TrimEnd('"');
```

Comment:
Seems like this can be just reduced to .Trim('"')?

Reply:
Done.